notesapi/v1/models.py

```diff
@@ -35,6 +35,6 @@ def create(cls, note_dict):
 
         note_dict['ranges'] = json.dumps(ranges)
         note_dict['user_id'] = note_dict.pop('user', None)
-        note_dict['tags'] = json.dumps(note_dict.get('tags', list()))
+        note_dict['tags'] = json.dumps(note_dict.get('tags', list()), ensure_ascii=False)
```

Comment:
`tags` in `note_dict` are already escaped with \uXXXX sequences. After applying `json.dumps` on tags list each unicode character is prefixed with another `\`, so `\u0441` becomes `\\u0441` which we don't want because if we write this data into elasticsearch, search will fail.